src/app/core/config/config-fix.ts

```diff
@@ -1018,7 +1018,7 @@ export const defaultJsonConfig = {
         label: $localize`:Label for the remarks about a dropout of a child:Dropout remarks`,
       },
       photo: {
-        dataType: "file",
+        dataType: "photo",
         label: $localize`:Label for the file field of a photo of a child:Photo`,
         editComponent: "EditPhoto",
```

Comment:
```suggestion
```
this should be automatically implied by the photo dataType, so can be removed here.